signal/src/max_abs.h

```diff
@@ -22,6 +22,10 @@ limitations under the License.
 namespace tflite {
 namespace tflm_signal {
 
+#if defined(XTENSA)
+int16_t XtensaMaxAbs16(const int16_t* input, int size);
```

Comment:
Can we just put this within MaxAbs16 so callers can just use that? They don't need to know they're calling an optimized version.

I'm thinking something like:

```
int16_t MaxAbs16(const int16_t* input, int size) {
#if defined(XTENSA)
  return XtensaMaxAbs16(input, size);
#else
  int16_t max = 0;
  ...
  return max;
#endif
}
```

WDYT?

Reply:
Yep that looks better, don't need to add a new function in the .h now too. Still keeping the name and separate function so its a bit cleaner.